tools/release/cherrypick_to_release_branch.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Script to cherry pick a commit to a 'in progress' release branch
+#
+#
+# Usage:
+# export RELEASE_BRANCH=axt_XXX_release_branch # if not already defined
+# bash tools/release/cherrypick_to_release_branch.sh commit_hash
+
+set -e
+
+if [[ -n $(git status -s) ]]; then
+    echo "Error: git directory is not clean; check 'git status'"
+    exit 1
+fi
+
+if [ "$#" -ne 1 ]; then
+    echo "Error: Unexpected number of parameters: Usage: cherrypick_to_release_branch.sh commit_hash"
+    exit 1
+fi
+
+if [[ ! "$RELEASE_BRANCH" =~ "release_branch" ]]; then
+    echo "Error: RELEASE_BRANCH env var is undefined or does not contain 'release_branch'"
```

Comment:
output the value to show which one?
```suggestion
    echo "Error: RELEASE_BRANCH env var (${RELEASE_BRANCH}) is undefined or does not contain 'release_branch'"
```